### PR TITLE
Added custom slide timing.

### PR DESCRIPTION
**Pull request by Bracciata**
**What**:

There is now the option to create a list of timings for the duration of each slide.

**Why**:

They allow users more freedom and assist with content such as slideshows where there is varying amounts of content per slide.

**How**:

There is a new Slider property called Interval list that allows the user to input that in addition to intervals and it will automatically apply the list instead.

**Checklist**:

- [x] Documentation added/updated
- [x] Typescript definitions updated
- [x] Tests added and passing
- [x] Ready to be merged

### CODE REVIEW

**Comment by Bracciata** [comment]
Fixing the Travis CLI build issues now

**Comment by Bracciata** [comment]
Bump

**Comment by Bracciata** [comment]
Reopen bump

File: src/Slider/Slider.jsx [hunk 1]
```diff
@@ -401,9 +404,22 @@ const Slider = class Slider extends React.Component {
...
   }
 
+
   play() {
-    const { playDirection } = this.props;
-    this.interval = setInterval(playDirection === 'forward' ? this.playForward : this.playBackward, this.props.interval);
+    if (this.props.intervalList) {
+      let priorIntervals = 0;
+      for (let i = 0; i < this.props.intervalList.length - 1; i += 1) {
+        setTimeout(this.props.playDirection === 'forward' ? this.playForward : this.playBackward, this.props.intervalList[i] + priorIntervals);
```
Comment (tim-steele) [inline src/Slider/Slider.jsx:412]: Is this setting a timeout for each interval listed in the array?

Comment (Bracciata) [reply src/Slider/Slider.jsx:412]: What it is doing is making the slides animate individually after a set amount of time that includes the sum of time waited for each prior slide.

**Pull request closed without merging**